src/sensors/softfusion/drivers/icm42688.h

```diff
@@ -113,7 +134,7 @@ struct ICM42688 {
 		static constexpr uint8_t FifoCount = 0x2e;
 		static constexpr uint8_t FifoData = 0x30;
 	};
-
+#if debug_icm42688_20bit
```

Comment:
Please define both structs as normal, but named FifoEntryAlignedDefault and FifoEntryAlignedHires or something similar, then you can create an alias with
```cpp
using FifoEntryAligned = std::conditional<DEBUG_ICM42688_HIRES, FifoEntryAlignedHires, FifoEntryAlignedDefault>;